src/shared-components/optionButton/__snapshots__/test.tsx.snap

```diff
@@ -264,9 +264,9 @@ exports[`<OptionButton /> UI snapshots radio unselected, with icon node prop 1`]
       <svg
         className="emotion-0"
         fill="none"
-        height={16}
+        height={48}
         viewBox="0 0 48 48"
-        width={16}
+        width={48}
```

Comment:
This is not a regression: we were passing in `AcneGlyph` without specifying `height`/`width` so it took on the default values of `16`.